staging/src/k8s.io/api/autoscaling/v2beta2/types.go

```diff
@@ -69,6 +69,9 @@ type HorizontalPodAutoscalerSpec struct {
 	// If not set, the default metric will be set to 80% average CPU utilization.
 	// +optional
 	Metrics []MetricSpec `json:"metrics,omitempty" protobuf:"bytes,4,rep,name=metrics"`
+	// constraints contain HPA object configuration parameters
+	// that configure each particular HPA scaling behaviour.
+	Constraints HorizontalPodAutoscalerScaleConstraints `json:"constraints,omitempty" protobuf:"bytes,5,opt,name=constraints"`
```

Comment:
I am not sure that 'constraints' is the best word here.  It's OK, but I want to ask if we considered an y others

Reply:
That is the best we have :-|, but we will happily consider an alternative.